worker-framework/docs/pages/en-us/Features.md

```diff
@@ -8,7 +8,7 @@ banner:
     subtitle: Analyze a Larger Range of Formats
```

Comment:
Has the documentation be re-built and re-published?  
  
This comment applies to all the document site content files.